letsencrypt/defaults.yaml

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 # vim: ft=yaml
 letsencrypt:
+  use_package: true
+  pkg: python-certbot-apache
```

Comment:
If the user also wants to install another plugin (e.g. _python3-certbot-nginx_), a dict and a call used by _pkg.installed:pkgs_ could be more flexible if I'm not mistaken.

Reply:
@aanriot I think this is a good Idea to use pkg.installed:pkgs !